scripts/test_sync_cargo_lock_version.py

```
@@ -0,0 +1,145 @@
+    def setUp(self) -> None:
+        self.root = pathlib.Path(__file__).resolve().parent.parent
+        if not (self.root / "Cargo.lock").is_file():
+            self.skipTest("Cargo.lock not present")
+
+    def test_only_workspace_members_change(self) -> None:
+        members = sclv.workspace_members(self.root / "Cargo.toml")
+        text = (self.root / "Cargo.lock").read_text()
+        out, _, seen = sclv.sync(text, members, "9.9.9")
+
+        self.assertEqual(members - seen, set(), "every member must be in the lockfile")
+        changed = sum(1 for a, b in zip(text.splitlines(), out.splitlines()) if a != b)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)